### PR TITLE
RDL, PP, CCA update p20200923/30

### PR DESCRIPTION
**Pull request by eruedin**

### CODE REVIEW

File: www/policy/PrivacyPolicy.html [hunk 1]
```diff
@@ -100,12 +101,7 @@ <h2 id="s10">10. Legal mandates</h2>
 </p>
 
 
-<p>If you need to contact us in writing, address your mail to:</p>
-<p>
-CAcert Inc.<br />
-PO Box 66 <br />
-Oatley NSW 2223<br />
-Australia
+<p>If you need to contact us in writing, address your mail to the postal address of CAcert Inc. The current postal address of CAcert Inc. can be found on CAcert's web site.
```
Comment (mcr) [inline www/policy/PrivacyPolicy.html:104]: I find this very circular: isn't this the CAcert web site?  Could we just link to where the address is then?

File: www/policy/CAcertCommunityAgreement.html [hunk 2]
```diff
@@ -295,8 +295,7 @@ <h3><a name="3">3.</a> Law and Jurisdiction</h3>
 
   <h4><a name="3.1">3.1</a> Governing Law</h4>
 
-  <p>This agreement is governed under the law of New South Wales, Australia,
-  being the home of the CAcert Inc. Association.</p>
+  <p>This agreement is governed under the law of New South Wales, Australia.</p>
```
Comment (mcr) [inline www/policy/CAcertCommunityAgreement.html:298]: Why would we still be governed under New South Wales law, and not Swiss law about Associations?

File: www/cap.html.php [hunk 3]
```diff
@@ -38,18 +38,18 @@
 	echo '<div style="text-align: right;">', "\n";
 	echo '<big><big><span style="font-weight: bold;">'._("Identity Verification Form (CAP) form").'</span></big></big><br>', "\n";
 	echo '</div>', "\n";
-	echo '<div style="text-align: right;">'.'Hangar 10 Airfield Avenue, Murwillumbah NSW 2484, New South Wales, (Commonwealth of) Australia  - <a href="http://www.cacert.org/"> http://www.cacert.org/</a><br></div>', "\n";
+	echo '<div style="text-align: right;">'.'Clos Belmont 2, 1208 Geneva, Switzerland  - <a href="http://www.cacert.org/"> http://www.cacert.org/</a><br></div>', "\n";
```
Comment (sat-hub) [inline www/cap.html.php:41]: Switzerland or Suisse?

File: www/policy/CAcertCommunityAgreement.html [hunk 4]
```diff
@@ -68,8 +69,7 @@ <h3><a name="0">0.</a> Introduction</h3>
   <h4><a name="0.1">0.1</a> Terms</h4>
 
   <ol>
-    <li>"CAcert" means CAcert Inc., a non-profit Association of Members
-    incorporated in New South Wales, Australia. Note that Association Members
+    <li>"CAcert" means CAcert Inc., a non-profit Association of Members. Note that Association Members
```
Comment (mcr) [inline www/policy/CAcertCommunityAgreement.html:72]: ... as defined by Swiss law <foo>?

File: www/capnew.php [hunk 5]
```diff
@@ -297,7 +297,7 @@
 define('LIME', '#C7FF00');  // RGB 199 255 0
 define('GREEN', '#00BE00'); // 0 190 0
 
-define('POBOX','Hangar 10 Airfield Avenue, Murwillumbah NSW 2484, New South Wales, (Commonwealth of) Australia');
+define('POBOX','Clos Belmont 2, 1208 Genève, Suisse');
```
Comment (sat-hub) [inline www/capnew.php:300]: Suisse or Switzerland?

**Review by mcr — CHANGES_REQUESTED** [review]

File: www/cap.html.php [hunk 6]
```diff
@@ -38,18 +38,18 @@
...
 
 	echo '<table border=1 cellspacing="0" cellpadding="0" bordercolor="lightblue" width="100%" style="color: white; background-color: rgb(112, 154, 186);" rules="groups">', "\n";
 	echo '<tbody>', "\n";
 	echo '<tr><td>', "\n";
 	echo '<tr>', "\n";
-	echo '    <td align="left"><font size=-7>'._("CAcert's Root Certificate sha256 fingerprints (since 2019)").'</font></td>', "\n";
-	echo '    <td align="right"><font size=-7>class 1: 07ED BD82 4A49 88CF EF42 15DA 20D4 8C2B 41D7 1529 D7C9 00F5 7092 6F27 7CC2 30C5</font></td>', "\n";
+	echo '    <td align="left"><font size=-7>'._("CAcert's Root Certificate sha256 fingerprints").'</font></td>', "\n";
+	echo '    <td align="right"><font size=-7>class 1 (2019): 07ED BD82 4A49 88CF EF42 15DA 20D4 8C2B 41D7 1529 D7C9 00F5 7092 6F27 7CC2 30C5</font></td>', "\n";
```
Comment (sat-hub) [inline www/cap.html.php:48]: ACK

File: www/cap.html.php [hunk 7]
```diff
@@ -38,18 +38,18 @@
...
 	echo '</tr>', "\n";
 	echo '<tr>', "\n";
 	echo '    <td></td>', "\n";
-	echo '    <td align="right"><font size=-7>class 3: F687 3D70 D675 96C2 ACBA 3440 1E69 738B 5270 1DD6 AB06 B497 49BC 5515 0936 D544</font></td>', "\n";
+	echo '    <td align="right"><font size=-7>class 3 (2021): 1BC5 A61A 2C0C 0132 C52B 284F 3DA0 D8DA CF71 7A0F 6C1D DF81 D80B 36EE E444 2869</font></td>', "\n";
```
Comment (sat-hub) [inline www/cap.html.php:52]: ACK

File: www/capnew.php [hunk 8]
```diff
@@ -313,11 +313,11 @@
 define('EXT','.html'); // default polciy doc extention, should be html
 /* finger print CAcert Root Key SHA256 since 2019*/ // should obtain this automatically
 define('CLASS1_SHA256','07ED BD82 4A49 88CF EF42 15DA 20D4 8C2B 41D7 1529 D7C9 00F5 7092 6F27 7CC2 30C5');
-define('CLASS3_SHA256','F687 3D70 D675 96C2 ACBA 3440 1E69 738B 5270 1DD6 AB06 B497 49BC 5515 0936 D544');
+define('CLASS3_SHA256','1BC5 A61A 2C0C 0132 C52B 284F 3DA0 D8DA CF71 7A0F 6C1D DF81 D80B 36EE E444 2869');
```
Comment (sat-hub) [inline www/capnew.php:316]: ACK

File: www/capnew.php [hunk 9]
```diff
@@ -313,11 +313,11 @@
...
 /* finger print CAcert Root Key */  // not to use since 2019
 /* finger print CAcert Root Key */ // should obtain this automatically
 define('CLASS1_SHA1','DDFC DA54 1E75 77AD DCA8 7E88 27A9 8A50 6032 52A5');
-define('CLASS3_SHA1','A7C4 8FBE 6B02 6DBD 0EC1 B465 B88D D813 EE1D EFA0');
+define('CLASS3_SHA1','D8A8 3A64 117F FD21 94FE E198 3DD2 5C7B 32A8 FFC8');
```
Comment (sat-hub) [inline www/capnew.php:320]: ACK

File: www/policy/PrivacyPolicy.html (hunk 1 above)
Comment (eruedin) [reply www/policy/PrivacyPolicy.html:104]: This is the text, voted by PolG

File: www/policy/CAcertCommunityAgreement.html (hunk 2 above)
Comment (eruedin) [reply www/policy/CAcertCommunityAgreement.html:298]: There are two different things:
- How we are governed internally: Common Law as before - nothing changed as a lot of internal rules, arbitration, etc. is based on this. Our internal rooles are as we want it (vote it at the AGM).
- The law on which the association is based an connected to world outside - this is ZGB/OR (Swiss law). Here, we cannot choose.

File: www/policy/CAcertCommunityAgreement.html (hunk 4 above)
Comment (eruedin) [reply www/policy/CAcertCommunityAgreement.html:72]: That is why we changed the CCA: From "Association based in xx, definied by yy law" to "association". That is more flexible for the association. Finally, for the community it does not matter if the association is based in this or another country. That is only relevant to the association's members and the committee.

File: www/cap.html.php (hunk 3 above)
Comment (eruedin) [reply www/cap.html.php:41]: if it is written in Englisch: Geneva, Switzerland.

File: www/capnew.php (hunk 5 above)
Comment (eruedin) [reply www/capnew.php:300]: If it is written in French: Genève, Suisse.
(actually, you could also write in German (Genf, Schweiz) or Italian (Ginevra, Svizzera))

**Comment by jandd** [comment]
This has been integrated in https://code.cacert.org/cacert/cacert-webdb/compare/v2024.09.21...v2024.10.01

**Pull request closed without merging**